### ISSUE TITLE
how long did you train the model?

### ISSUE THREAD

**Issue by lseiyjg** [nagadomi/waifu2x#235]
I have started a SR network like upconv_7 recently in tensorflow. I cut 5000 high resolution images into about 500,000 pieces of 256x256 training images. It cost me about 2 days for 1 epoch to train these data.  If the training speed is right? Or I will focus on optimizing my code to speed up the training stage.

**Comment by nagadomi** [comment]
I do not remember exactly, but training is finished in around one day.
However, waifu2x has trained total 7 models for noise levels, so it will take a week to finish all.

and, waifu2x uses 24x24 randomly selected 64 patches from 1 image for each epoch. Not all image areas are used in 1 epoch.

**Comment by lseiyjg** [comment]
thanks！

**Comment by yu45020** [comment]
No need to train the model from scratch. Using the pre-trained model's weights to initialize your model will save you time as the loss will converge quickly (hopefully).
